src/EventGridEmulator/Network/SubscriberPolicyHttpMessageHandler.cs

```
@@ -78,6 +78,7 @@ protected override async Task<HttpResponseMessage> SendCoreAsync(HttpRequestMess
         }
         catch (HttpRequestException ex)
         {
+            this._logger.LogInformation(ex, "*** More information ***");
```

Do we need a TODO here?

---

I will change it to a debug. That error was swallowed and there were no clear trace on what was failing.